src/generalizedlinearmixedmodel.jl

```diff
@@ -269,16 +274,19 @@ function fit!(
         optsum.final = copy(optsum.initial)
     end
     setpar! = fast ? setθ! : setβθ!
+    prog = ProgressUnknown("Minimizing"; showspeed=true)
     function obj(x, g)
         isempty(g) || throw(ArgumentError("g should be empty for this objective"))
         val = deviance(pirls!(setpar!(m, x), fast, verbose), nAGQ)
         verbose && println(round(val, digits = 5), " ", x)
```

Comment:
```suggestion
```

Reply:
I initially thought we should remove this, but I think it actually makes sense to keep it in and have `verbose` and `progress` be completely independent for GLMM. `progress` corresponds to `progress in LMM, while `verbose` is well, exceptionally verbose and mostly interesting for diagnosing problems.